qa/rpc-tests/wallet_accounts.py

```diff
@@ -58,16 +63,21 @@ def run_test(self):
         # Verify that just creating the account does not generate any visible addresses
         addresses = self.nodes[0].z_listaddresses()
         assert_equal([], addresses)
+        accounts = self.nodes[0].z_listaccounts()
+        assert_equal(accounts[0]['account'], 0)
```

Comment:
This and the checks below should probably check the length of `accounts`, to make sure there are no spurious ones.